src/Controller/CaseController.php

```diff
@@ -38,10 +42,60 @@ public function summary(CaseEntity $case): Response
     /**
      * @Route("/{id}/information", name="case_information", methods={"GET"})
      */
-    public function information(CaseEntity $case): Response
+    public function information(CaseEntity $case, ComplaintCategoryRepository $categoryRepository): Response
     {
+        /** @var ComplaintCategory $complaintCategory */
+        $complaintCategory = $categoryRepository->findOneBy([
+            'board' => $case->getBoard(),
+            'municipality' => $case->getMunicipality(),
+            'name' => $case->getCaseType(),
+        ]);
+
         return $this->render('case/information.html.twig', [
             'case' => $case,
+            'complaintCategory' => $complaintCategory,
+        ]);
+    }
+
+    /**
+     * @Route("/{id}/information/edit", name="case_information_edit", methods={"GET", "POST"})
+     */
+    public function editInformation(CaseEntity $case, ComplaintCategoryRepository $categoryRepository, Request $request): Response
```

Comment:
Same as above with the showInformation method. This method should be renamed to `edit`and the route should be `@Route("/{id}/edit)" and the route name should be `case_edit`.

Reply:
Renamed route and function accordingly.